package.json

```
@@ -16,32 +16,32 @@
     }
   },
   "dependencies": {
-    "firebase-admin": "^8.12.1",
-    "firebase-functions": "^3.6.2",
-    "next": "^9.4.4",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "styled-components": "^5.1.1"
+    "firebase-admin": "^9.3.0",
+    "firebase-functions": "^3.11.0",
+    "next": "^10.0.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "styled-components": "^5.2.1"
   },
   "devDependencies": {
-    "@types/react": "^16.9.35",
-    "@types/react-dom": "^16.9.8",
-    "@types/styled-components": "^5.1.0",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
+    "@types/react": "^16.9.56",
+    "@types/react-dom": "^16.9.9",
+    "@types/styled-components": "^5.1.4",
+    "@typescript-eslint/eslint-plugin": "^4.6.1",
     "@typescript-eslint/parser": "^2.34.0",
```

parserの方もバージョン上げないと最新のTypeScript に対応してくれなかったりするのでこっちも上げてもらえると助かります :bow:
```suggestion
    "@typescript-eslint/parser": "^4.6.1",
```

---

漏れてました :pray: